packages/antd/src/templates/BaseInputTemplate/index.tsx

```diff
@@ -10,6 +10,7 @@ import {
   RJSFSchema,
   StrictRJSFSchema,
 } from '@rjsf/utils';
+import ClearButton from '../IconButton';
```

Comment:
Thanks for making a separate button... Now you'll need to get it directly from the registry rather than importing it. This needs to be done in all the themes

Something like:
```
  const { ClearButton } = registry.templates.ButtonTemplates;

```